src/components/SettingDialog.vue

```diff
@@ -169,25 +276,67 @@ export default defineComponent({
       } else change();
     };
 
-    const fileEncoding = computed({
-      get: () => store.state.fileEncoding,
-      set: (encoding: Encoding) =>
-        store.dispatch(SET_FILE_ENCODING, {
-          encoding: encoding,
-        }),
+    const restartEngineProcess = () => {
+      store.dispatch(RESTART_ENGINE);
+    };
+
+    const savingSetting = computed(() => store.state.savingSetting);
+
+    const handleSavingSettingChange = (key: string, data: string | boolean) => {
+      if (key === "fixedExportDir") {
+        inputCheckDirExists(data.toString()).then((res) => {
+          if (res) {
+            store.dispatch(SET_SAVING_SETTING_DATA, {
+              data: { ...savingSetting.value, [key]: data },
+            });
+          }
+        });
+      } else {
+        store.dispatch(SET_SAVING_SETTING_DATA, {
+          data: { ...savingSetting.value, [key]: data },
+        });
+      }
+    };
+
+    const openFileExplore = async () => {
+      const path = await window.electron.showOpenDirectoryDialog({
+        title: "デフォルトのフォルダを選択",
```

Comment:
```suggestion
        title: "書き出し先のフォルダを選択",
```

このようにサジェストしてもいいですか？
適用ボタンを押せば修正されてコミットされるはずです。

Can I suggest it this way?
Just click the Apply button and it should be modified and committed.